upgrader/apps/upgrader/lib/utility/UpgradeUtility.php

```diff
@@ -382,10 +382,22 @@ public function getNotes($startIncNumber, $endIncNumber) {
     }
 
     public function getInstalledAddons() {
+        if (!$this->tableExists('ohrm_marketplace_addon')) {
+            return [];
+        }
         $query = 'select * from ohrm_marketplace_addon where plugin_name is not null';
         return mysqli_fetch_all($this->executeSql($query), MYSQLI_ASSOC);
     }
 
+    public function tableExists($tableName) {
```

Comment:
seems this function check table exits + non zero records.. better to rename the function to reflect that 
something like tableExistsWithData or nonEmptyTableExists